packages/xo-web/src/common/xo/multipathing-modal/index.js

```diff
@@ -0,0 +1,52 @@
+import _ from '../../intl'
+import Collapse from '../../collapse'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { Sr } from '../../render-xo-item'
+import { connectStore } from '../../utils'
+import { createGetObjectsOfType, createSelector } from '../../selectors'
+
+@connectStore(
+  {
+    srIds: createSelector(
+      createGetObjectsOfType('PBD').filter((_, { hostIds }) => pbd =>
+        hostIds.includes(pbd.host)
+      ),
+      pbds => {
+        const srIds = new Set([])
+        for (const id in pbds) {
+          srIds.add(pbds[id].SR)
+        }
+        return [...srIds]
```

Comment:
Why not a simple `map`?

Reply:
Importing a module globally from `common` can cause troubles if a dependence import a module with the same name.